_includes/cfp.html

```diff
@@ -0,0 +1,33 @@
+<div id="cfp" class="page-wrapper cfp-section">
+  <div class="page-details">
+    <div class="title">Call  for Proposals</div>
+    <div class="description">
+      Technical talks are the most important event at PyCon India,
+      the core of the conference essentially. Two of the four days
+      are dedicated to talks. We are all set to go virtual.
```

Comment:
You always wanted to speak in public or  wanted to inspire the next generation of audience here is an opportunity for you , we have been accommodating every level of talks more so ever beginners is encourages to submit one. This year being a virtual conference it gives us a  unique opportunity to practice and share your videos for the CFP

Reply:
Let's create a PR out of this so everyone can comment. cc: @vipulgupta2048

Reply:
Created an issue to track the discussion: https://github.com/pythonindia/inpycon2020/issues/20